src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -127,7 +120,7 @@ public Down(Address address)
     /// <summary>
     /// Command to join the cluster. Sent when a node wants to join another node (the receiver).
     /// </summary>
-    internal sealed class InternalClusterAction
+    internal static class InternalClusterAction
```

Comment:
This class is never instantiated, should be made `static` to resolve some compiler grumbling.